enigma-core/app/src/esgx/ocalls_u.rs

```diff
@@ -197,6 +198,32 @@ pub unsafe extern "C" fn ocall_get_deltas(db_ptr: *const RawPointer, addr: &Cont
     }
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn ocall_remove_delta(db_ptr: *const RawPointer,
+                                            contract_address: &ContractAddress, _delta_index: *const u32) -> EnclaveReturn {
+    let delta_index = ptr::read(_delta_index);
+    let key = DeltaKey::new(*contract_address, Stype::Delta(delta_index));
+    let db: &mut DB = match (*db_ptr).get_mut_ref() {
+        Ok(db) => db,
+        Err(e) => {
+            error!("{}", e);
+            return EnclaveReturn::OcallDBError
+        }
+    };
+    match db.delete(&key) {
+        Ok(_) => EnclaveReturn::Success,
+        Err(e) => {
+            match e.downcast::<DBErr>() {
+                Ok(_) =>  EnclaveReturn::Success,
+                Err(_) => {
+                    println!("Failed removing delta: {:?}", &key);
```

Comment:
I think that `error!` should replace`println!` here. Isn't it worth to print `e` as part of the message?